docs/reference/ingest/apis/get-pipeline.asciidoc

```diff
@@ -46,73 +86,51 @@ Example response:
     ]
   }
 }
---------------------------------------------------
+----
 
-For each returned pipeline, the source and the version are returned.
-The version is useful for knowing which version of the pipeline the node has.
-You can specify multiple IDs to return more than one pipeline. Wildcards are also supported.
 
-[float]
-[[versioning-pipelines]]
```

Comment:
Moved to put pipeline API docs:
https://github.com/elastic/elasticsearch/compare/master...jrodewig:reformat.get-pipeline#diff-a54ad19e7f8ba7cc253f32a775128d1bR23